src/components/data/data-utils.js

```diff
@@ -36,6 +36,25 @@ const errorTextStyle = { color: colors.danger(), fontWeight: 'bold', fontSize: 1
 
 export const parseGsUri = uri => _.drop(1, /gs:[/][/]([^/]+)[/](.+)/.exec(uri))
 
+export const getDownloadCommand = (fileName, gsUri, accessUrl) => {
+  if (accessUrl && accessUrl.url) {
+    const headers = _.flow(
+      _.toPairs,
+      _.reduce(
+        (headers, [header, value]) => {
+          return `${headers}--header='${header}: ${value}' `
+        },
+        ''
+      )
+    )(accessUrl.headers || {})
```

Comment:
`_.toPairs` is guarded against undefined and other falsy values, so you shouldn't have to provide an alternative here.

Reply:
Hunh. I saw `guarded` in the docs and had know idea what it meant. Thanks!